renovate.json

```diff
@@ -7,6 +7,7 @@
     ":ignoreUnstable",
     ":prImmediately",
     ":semanticCommitsDisabled",
+    ":updateNotScheduled",
```

Comment:
**issue**: it looks like [this option](https://docs.renovatebot.com/configuration-options/#updatenotscheduled) will update existing PRs, but not allow new Renovate PRs to be created. i think we should remove the `schedule:nonOfficeHours` (and this option) instead, since according to [the docs](https://docs.renovatebot.com/configuration-options/#schedule):

> The default value for schedule is "at any time", which is functionally the same as declaring a null schedule. i.e. Renovate will run on the repository around the clock.

Reply:
so I checked and you were right, there were several attempts to make renovate only open PRs out of hours, but going back in history to 2 years ago, `schedule:nonOfficeHours` is not in the config and `updateNotScheduled` is. The docs say that `updateNotScheduled` defaults to `true`, so I am thinking that removing it or keeping it in the config would turn out in the same behaviour?

Reply:
my interpretation of the docs is that `updateNotScheduled` won't have any effect when there's no schedule. i think we should remove it anyway to avoid confusion